CtvReferenceApp/src/main/java/com/truex/ctv/referenceapp/ads/AdPodManager.java

```diff
@@ -0,0 +1,358 @@
+package com.truex.ctv.referenceapp.ads;
+
+import android.content.Context;
+import android.util.Log;
+import android.view.ViewGroup;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import androidx.annotation.OptIn;
+import androidx.media3.common.MediaItem;
+import androidx.media3.common.util.UnstableApi;
+import androidx.media3.exoplayer.source.ConcatenatingMediaSource2;
+import androidx.media3.exoplayer.source.MediaSource;
+import androidx.media3.exoplayer.source.ProgressiveMediaSource;
+import androidx.media3.datasource.DataSource;
+
+public class AdPodManager {
```

Comment:
rename to AdManager?
it then conceptually managers ad breaks and ads in general